### ISSUE TITLE
Web admin : identifier domain with dash clears your domain form

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#3311]
In Policies and Access Control -> Active Directory Domains -> Add domain

If you define a new domain identifier and use a dash in `Identifier` field, all your form will be clear when saving configuration. Really annoying if you filled all the form.

### Expected results
- [ ] Warn user about forbidden characters in `Identifier` field
- [ ] Keep form unmodified if forbidden characters are found

To sum up : behavior should be equivalent to `Profile name` in Connection Profiles
